class Exception {
  constructor() {
  }
  
  showMessage(message) {
    alert(message);
  }
}
